From c71686d040968fef9d0ffef3a160f65dbd0db9da Mon Sep 17 00:00:00 2001
From: Michel Pollet <buserror@gmail.com>
Date: Wed, 28 Nov 2012 16:47:40 +0000
Subject: [PATCH 08/11] imx23-olinuxino: Add 16 bits LCD declaration

Added a block for (dumb) 16 bits TFT to the
device tree. Since it clashes with the
pin declaration for the SPI, both blocks
are disabled by default and require custom
editing (or bootloader) to activate.

Signed-off-by: Claude Schwarz <claude.schwarz@gmail.com>
Signed-off-by: Michel Pollet <buserror@gmail.com>
---
 arch/arm/boot/dts/imx23-olinuxino.dts |   12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx23-olinuxino.dts b/arch/arm/boot/dts/imx23-olinuxino.dts
index e5b080d..35fc9a5 100644
--- a/arch/arm/boot/dts/imx23-olinuxino.dts
+++ b/arch/arm/boot/dts/imx23-olinuxino.dts
@@ -30,7 +30,7 @@
 				bus-num = <0>;
 				pinctrl-names = "default";
 				pinctrl-0 = <&spi1_control_pins &spi1_cs_pins>;
-				status = "okay";
+				status = "disabled";
 
 				#address-cells = <1>;
 				#size-cells = <0>;
@@ -65,7 +65,13 @@
 					fsl,pull-up = <1>;
 				};
 			};
-			
+			lcdif@80030000 {
+				pinctrl-names = "default";
+				pinctrl-0 = <&lcdif_16bit_pins_a>;
+				/*   panel-enable-gpios = <&gpio1 18 0>; */
+				status = "disabled";
+			};
+
 			ssp0: ssp@80010000 {
 				compatible = "fsl,imx23-mmc";
 				pinctrl-names = "default";
@@ -82,7 +88,7 @@
 					reg = <0>;
 					fsl,pinmux-ids = <
 						0x2013 /* MX23_PAD_SSP1_DETECT__GPIO_2_1 */
-						0x0113 /* MX23_PAD_GPMI_ALE__GPIO_0_17 */
+					//	0x0113 /* MX23_PAD_GPMI_ALE__GPIO_0_17 */
 					>;
 					fsl,drive-strength = <0>;
 					fsl,voltage = <1>;
-- 
1.7.10.4

